### ISSUE TITLE
Would you like to bypass splash screen?

### ISSUE THREAD

**Issue by RealMove** [RenardDev/OreansCrack#3]
I've really loved your work & wondering if you can bypass demo splash screen to, its too annoying.

**Comment by RenardDev** [comment]
I'm still thinking about what to use for that. I'll leave this issue open until I do.

**Comment by RealMove** [comment]
I hope you can make this :) Good luck

29 Haz 2023 Per, saat 11:33 tarihinde Ren ***@***.***> şunu
yazdı:

**Comment by aydynx** [comment]
Hey, im having this problem too where it doesnt bypass the demo splash screen for themida and winlicense, but everything works perfectly with code virtualizer.

**Comment by RealMove** [comment]
> Hey, im having this problem too where it doesnt bypass the demo splash screen for themida and winlicense, but everything works perfectly with code virtualizer.

Nah, the reason it works with code virtualizer is code virtualizer doesn't have a demo splash screen. I'm still using this app and I hope Renard is gonna do it soon :)

**Comment by JewFrags** [comment]
any future work planned in regards to this or no?

**Comment by RenardDev** [comment]
> any future work planned in regards to this or no?

At the moment it is impossible to competently perform this with the existing [Detours](https://github.com/RenardDev/Detours). Another project is currently being developed, which may give a different direction for solving this task (it will be something similar to [Blackbone](https://github.com/DarthTon/Blackbone))
